### ISSUE TITLE
No way to reset an offset

### ISSUE THREAD

**Issue by rjgrazioli** [matthewsimo/scss-flex-grid#14]
I'm setting my `col` to `.off-sm-5` and would like to remove the offset on a larger screen. Would a `.off-md-0` make sense here?

**Comment by matthewsimo** [comment]
You can pretty easily create a `.offset-reset` class that sets `margin-left` to zero and apply it to the element. I'm going to put this on hold and if enough folks I don't see harm in adding it in.

**Comment by timogauweiler** [comment]
Got exactly the same issue and had the same thoughts as @rjgrazioli. `.off-md-0` or `.off-md-reset` (with a media query class) would make more sense to me.

**Comment by timdiacon** [comment]
+1 from me

`.off-md-0`

Would make sense to me 👍

**Comment by matthewsimo** [comment]
Hey - quick update on this front. I got a working solution in place but I'm still testing it. I went with `.($fg-class-off)-(namespace)-reset` as a naming convention since it resets any offset for that or larger namespaces due to the order of namespace declaration (same behavior as declaring column span on smaller applies to larger unless overridden).

In any case, I plan on updating docs, finish testing and cutting a release tonight. Thanks!
